### PR TITLE
MSPSDS-982: Update documentation surrounding database setup

### PR DESCRIPTION
**Pull request by broder**
This doesn't need to go in before the production release, it's mostly just for future reference.

### CODE REVIEW

File: keycloak/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ To create a Keycloak database for the current space:
 
     cf marketplace -s postgres
     cf enable-service-access postgres
-    cf create-service postgres tiny-unencrypted-10.5 keycloak-database
+    cf create-service postgres small-10.5 keycloak-database
```
Comment (jasiekmiko) [inline keycloak/README.md]: Just to clarify that we've moved all envs to this size?

File: mspsds-web/README.md [hunk 2]
```diff
@@ -130,11 +128,8 @@ To create an Elasticsearch instance for the current space:
 To create a redis instance for the current space. 
 
     cf marketplace -s redis
-    cf create-service redis tiny-3.2 mspsds-redis-queue
-    cf create-service redis tiny-3.2 mspsds-redis-session
-
-Larger options should be considered if required.
-The current worker (sidekiq), which uses mspsds-redis-queue only works with the unclustered version.
```
Comment (jasiekmiko) [inline mspsds-web/README.md]: do we know that the unclustered clause no longer apply?

File: keycloak/README.md (hunk 1 above)
Comment (broder) [reply keycloak/README.md]: We have now :)

File: mspsds-web/README.md (hunk 2 above)
Comment (broder) [reply mspsds-web/README.md]: No, I had assumed that they removed the option for unclustered instances on PaaS and therefore, it was working with a clustered version. `tiny-3.2` is a single node so shouldn't actually be doing any clustering. The other size available (`medium-ha-3.2`) does have multiple nodes so might be clustered, so we probably should keep the warning.

**Pull request merged**